qmarkdowntextedit.cpp

```diff
@@ -303,6 +304,10 @@ bool QMarkdownTextEdit::eventFilter(QObject *obj, QEvent *event) {
             _searchWidget->doSearch(
                     !keyEvent->modifiers().testFlag(Qt::ShiftModifier));
             return true;
+        } else if ((keyEvent->key() == Qt::Key_Z) &&
+                   (keyEvent->modifiers().testFlag(Qt::ControlModifier))) {
```

Comment:
I hope that does it's thing on macOS :grin:

Reply:
yes they have the command thing. Isn't it the same? maybe not. :thinking:

Reply:
Ctrl should be automatically be mapped to Cmd, we'll see :smile: